libraries/AP_HAL_Linux/RCInput_ZYNQ.cpp

```diff
@@ -32,10 +32,10 @@ void RCInput_ZYNQ::init()
     if (mem_fd == -1) {
         AP_HAL::panic("Unable to open /dev/mem");
     }
-    pulse_input = (volatile uint32_t*) mmap(0, 0x1000, PROT_READ|PROT_WRITE,
+
+    pulse_input = (volatile uint32_t*) mmap(NULL, 0x1000, PROT_READ|PROT_WRITE,
```

Comment:
Forgive my ignorance, but what's the advantage to switching to NULL here?

Reply:
Since it's an address, both are NULL, just one is more verbose.